pkg/domain/infra/abi/images.go

```diff
@@ -720,7 +720,10 @@ func (ir *ImageEngine) Sign(ctx context.Context, names []string, options entitie
 	return nil, nil
 }
 
-func getSigFilename(sigStoreDirPath string) (string, error) {
+func getSigFilename(sigStoreDirPath string, options entities.SignOptions) (string, error) {
+	if options.AuthFile != "" {
+		return options.AuthFile, nil
```

Comment:
Is this using the `--authfile` option that usually points to auth.json, with registry login credentials, as a destination for where to create the signature file, with a completely different file format?

The test seems to confirm that it is.

How does that make sense, _especially_ if it is, as the man page change suggests, attached to the registry variable? Will every `podman image sign` now overwrite auth.json with a signature?

Reply:
Your right, the option should be sigfile not authfile.

Reply:
@mtrmac Take a look now, I switched from --authfile to --sigfile.  I don't think authfile is involved in image signing.